roofit/hs3/src/JSONFactories_HistFactory.cxx

```diff
@@ -461,7 +462,7 @@ class RooRealSumPdfFactory : public RooFit::JSONIO::Importer {
       std::vector<std::string> funcnames;
       std::vector<std::string> coefnames;
       RooArgSet observables;
-      if (p.has_child("observables")) {
```

Comment:
Why are you changing from observables to variables ?
I find that the name observables reflect better the content

Reply:
Good question, I remember that was a consensus from the last discussion in the HS3 meeting that Carsten organized, but it would be good to note this down again here.

@cburgard, can you comment?

Reply:
Here are the two main arguments from what I can recall of the discussion:
1. Different communities have a different conception of what an "observable" is. E.g., most theorists will call "observables" what we in RooFit call "parameters of interest", thus we opted for not using the word "observable" at all because it has very different connotations in different communities. Also, "observables" really only makes sense in a scenario where you are dealing with real data - if you are building models mostly for the purpose of generating expected distributions, nothing is "observed".
2. From a statistics point of view, what we call "observables" are the variates of the distributions, so "variables" makes most sense from a purely mathematical point of view.

Reply:
Thanks Carsten!